pkg/providers/amifamily/ami.go

```diff
@@ -219,11 +219,22 @@ func (p *Provider) fetchAMIsFromEC2(ctx context.Context, amiSelector map[string]
 	p.ec2Cache.SetDefault(fmt.Sprint(hash), output.Images)
 	amiIDs := lo.Map(output.Images, func(ami *ec2.Image, _ int) string { return *ami.ImageId })
 	if p.cm.HasChanged("amiIDs", amiIDs) {
-		logging.FromContext(ctx).With("ami-ids", amiIDs).Debugf("discovered images")
+		logging.FromContext(ctx).With("ids", concatenateAmiIDs(amiIDs), "count", len(amiIDs)).Debugf("discovered images")
 	}
 	return output.Images, nil
 }
 
+func concatenateAmiIDs(amisIds []string) string {
+	var amiString string
+	if len(amisIds) > 5 {
```

Comment:
IMO I think there are plenty of valid use cases to have more than 5 distinct AMIs that are used. Can we stick with 25 here and reduce it later if we feel like it's still too noisy?